### PR TITLE
Hdxdsys 801 add parameter tables

### PR DESCRIPTION
**Pull request by turnerm**
Current issues:
- Lots of repetition in the YAMLs. Will be difficult to keep it in sync with the API, I should perhaps write a script that checks that the files are in sync with the returned fields.
- The tables are quite narrow and the parameter names get wrapped, this doesn't help with readability

### CODE REVIEW

File: docs/data_usage_guides/enum_parameters/population_group_parameters.yaml [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
+- Population Group: NAT
+  Description: 'naturalized refugees'
+- Population Group: RDP
+  Description: 'returned IDPs'
+- Population Group: POC
+  Description: 'persons of concern'
+- Population Group: RRI
+  Description: >
+   An umbrella term for returnees, used in the humanitarian
+   needs data which not distinguish between returned refugees and returned IDPs
```
Comment (mcarans) [inline docs/data_usage_guides/enum_parameters/population_group_parameters.yaml]: which does not

**Review by mcarans — APPROVED** [review]
Looks fine to me

File: docs/data_usage_guides/endpoint_parameters/dataset_parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+- Parameter: '`hdx_id`'
```
Comment (alexandru-m-g) [inline docs/data_usage_guides/endpoint_parameters/dataset_parameters.yaml]: Some field names have changed here: https://github.com/OCHA-DAP/hdx-hapi/pull/133 
hdx_id -> dataset_hdx_id
hdx_stub - > dataset_hdx_stub
title -> dataset_hdx_title

File: docs/data_usage_guides/endpoint_parameters/resource_parameters.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
+- Parameter: '`hdx_id`'
```
Comment (alexandru-m-g) [inline docs/data_usage_guides/endpoint_parameters/resource_parameters.yaml]: Similar to the dataset endpoint, some field names have changes here as well: https://github.com/OCHA-DAP/hdx-hapi/pull/133

hdx_id -> resource_hdx_id
dataset_title -> dataset_hdx_title

**Review by alexandru-m-g — APPROVED** [review]

**Pull request merged**